package.json

```diff
@@ -80,6 +80,7 @@
       "@types/node": "^18.16.16",
       "axios": "1.6.7",
       "chokidar": "3.5.2",
+      "ip": "2.0.1",
```

Comment:
Does this need to be in `package.json`? We don't seem to use internally it. Would it be enough to bump only `pnpm-lock.yaml`?

Reply:
this is in the `pnpm.overrides` block. Unfortunately pnpm doesn't have a way to define this block in another file.

Reply:
Oh right, thought it was the `dependencies` block. Then all good 👌